core/src/scene/sceneLoader.cpp

```diff
@@ -64,6 +65,97 @@ void printFilters(const SceneLayer& layer, int indent){
     }
 };
 
+void SceneLoader::applyGlobalProperties(Node& node, Scene& scene) {
+    switch(node.Type()) {
+    case NodeType::Scalar:
+        {
+            std::string key = node.Scalar();
+            const std::string wildcard = "_$_";
```

Comment:
Weird.. Can you link me?

That seems like a poor choice, since those characters aren't part of structural yaml syntax they seem more likely to be used in a member name than `:`.